go.mod

```diff
@@ -1,7 +1,9 @@
 module github.com/nats-io/nats-server/v2
 
+go 1.14
```

Comment:
I would leave this out for now

Reply:
Currently this is in there to make travis happy as I'll have to wait for the corresponding jwt pr to be merged.
That however is contingent on this being approved.
So, once the jwt pr is comitted I'll regenerate go.mod without and squash.
I'll make sure the go version is not changed.